doc/ref_model/chapters/appendix-a.md

```diff
@@ -66,6 +68,18 @@ Without arguing for or against SR-IOV, CNTT provides the following anecdotes whi
 
 **CPU Instruction Sets and Extensions:**
 <content needed>
+  
+**Raw Disk Device Mapping**
+<content needed>
+Overview of technology:
+
+It is possible to map SAN LUNs or local disks to a VM, giving direct hardware access to the specified device.  Given the state of today's storage technology, perhaps CNTT should not provide an exception for this.
```

Comment:
I thought that ideally we shouldn't be talking about VMs at RM level - needs to be agnostic to bare metal, VMs, containers or whatever?

Reply:
Sorry, that was a quick write up to give an overview and meant to start the conversation of should we call this out as explicitly not supported.  Perhaps the use of C/VNF instead of VM?

Reply:
OK, understood.  Yes, I think something more agnostic like that would be good.

Reply:
Do we have a preferred term yet?  ETSI uses Virtual Deployment Unit (VDU) but I think it's the use of the term Virtual that we are concerned about?